### ISSUE TITLE
Exception

### ISSUE THREAD

**Issue by 0x0a11c0de** [nielsfaber/alarmo#157]
I had this in my logs this morning:

```
Traceback (most recent call last):
  File "/config/custom_components/alarmo/__init__.py", line 272, in async_handle_event
    elif event.data["action"]:
KeyError: 'action'
```

**Comment by nielsfaber** [comment]
Should be solved with https://github.com/nielsfaber/alarmo/commit/dc133c6b9834b221337586b6245ed3d3a9495e50

**Comment by nielsfaber** [comment]
Solved in v1.5.5
